README.md

```diff
@@ -1,470 +1,74 @@
 [![Code style: black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/psf/black)
 ![Tests](https://github.com/Breakthrough-Energy/PowerSimData/workflows/Pytest/badge.svg)
+[![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT)
 
 
 
 # PowerSimData
-This package has been written in order to carry out power flow study in the U.S. electrical grid. This framework allows the user to easily build extensive scenarios.
+**PowerSimData** is part of a Python software ecosystem developed by [Breakthrough
+Energy Sciences](https://science.breakthroughenergy.org/) to carry out power flow study
+in the U.S. electrical grid.
 
-PowerSimData is part of a set of packages representing Breakthrough Energy's power system model. More information regarding the installation of the model as well as the contribution guide can be found [here](https://breakthrough-energy.github.io/docs/).
 
+## Main Features
+Here are a few things that **PowerSimData** can do:
+* Provide a flexible modeling to create complex scenarios
+* Perform investment cost studies
+* Call simulation engine of power system operation to run power flow study
```

Comment:
Might be clearer to say something like "Run power flow study using interface to external simulation engine"